plugins/base/src/main/kotlin/translators/documentables/DescriptionSections.kt

```diff
@@ -0,0 +1,320 @@
+        sourceSets = availableSourceSets
+    )
+    {
+        availableSourceSets.forEach { sourceSet ->
+            val possibleFallbacks = availableSourceSets.getPossibleFallback(sourceSet)
+            seeAlsoTags.forEach { (_, see) ->
+                (see[sourceSet] ?: see.fallback(possibleFallbacks))?.let { seeTag ->
+                    row(
+                        sourceSets = setOf(sourceSet),
+                        kind = ContentKind.Comment
+                    ) {
+                        seeTag.address?.let { dri ->
+                            link(
+                                text = seeTag.name.removePrefix("${dri.packageName}."),
+                                address = dri,
+                                kind = ContentKind.Comment,
+                                styles = mainStyles + ContentStyle.RowTitle
+                            )
+                        } ?: text(
+                            text = seeTag.name,
+                            kind = ContentKind.Comment,
+                            styles = mainStyles + ContentStyle.RowTitle
+                        )
+                        if (seeTag.isNotEmpty()) {
+                            comment(seeTag.root)
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+internal fun PageContentBuilder.DocumentableContentBuilder.throwsSectionContent(tags: GroupedTags) {
+    val throwsTags = tags.withTypeNamed<Throws>() ?: return
+    val availableSourceSets = throwsTags.availableSourceSets()
+
+    header(KDOC_TAG_HEADER_LEVEL, "Throws", sourceSets = availableSourceSets)
+    table(
+        kind = ContentKind.Main,
+        sourceSets = availableSourceSets,
+        extra = mainExtra + SimpleAttr.header("Throws")
+    ) {
+        throwsTags.forEach { (throwsName, throwsPerSourceSet) ->
+            throwsPerSourceSet.forEach { (sourceSet, throws) ->
+                row(sourceSets = setOf(sourceSet)) {
+                    group(styles = mainStyles + ContentStyle.RowTitle) {
+                        throws.exceptionAddress?.let {
+                            val className = it.takeIf { it.target is PointingToDeclaration }?.classNames
+                            link(text = className ?: throwsName, address = it)
+                        } ?: text(throwsName)
+                    }
+                    if (throws.isNotEmpty()) {
+                        comment(throws.root)
+                    }
+                }
+            }
+        }
+    }
+}
+
+internal fun PageContentBuilder.DocumentableContentBuilder.samplesSectionContent(tags: GroupedTags) {
+    val samples = tags.withTypeNamed<Sample>() ?: return
+    val availableSourceSets = samples.availableSourceSets()
+
+    header(KDOC_TAG_HEADER_LEVEL, "Samples", kind = ContentKind.Sample, sourceSets = availableSourceSets)
+    availableSourceSets.forEach { sourceSet ->
+        group(
+            sourceSets = setOf(sourceSet),
```

Comment:
`sourceSets = availableSourceSets`?

Reply:
In that case, for MPP all samples will be shown on each platform tab